EIPS/eip-7577.md

```diff
@@ -0,0 +1,110 @@
+
+This EIP proposes a semantic versioning scheme and an addition of a CHANGELOG section for EIPs that enables clearer communication within the community and allows the scope of a change to be ascertained at first glance. Furthermore, client implementation and testing toolchains can query EIP changes and automatically flag incompatibilities between the EIP's current specification and between client and test implementations.
+
+## Specification
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119 and RFC 8174.
+
+Once an EIP has moved out of "Draft" status, it MUST use the EIP versioning scheme outlined below. It MAY already use the versioning scheme in "Draft" status, which could be useful if the specification is actively being implemented. If more than one team is implementing the specification, it is RECOMMENDED to change the EIP's status to "Review".
+
+The EIP versioning scheme MUST apply the following semantic versioning scheme of `MAJOR.MINOR.PATCH`, based on [Semantic Versioning 2.0.0](../assets/eip-7577/semver.md):
+
+1. `MAJOR`: A breaking change to the specifications that requires an implementation change and a change to the reference tests.
+2. `MINOR`: An addition to the specifications that requires additional implementation and additional test coverage to be added to the reference tests.
+3. `PATCH`: Any cosmetic change to, or a reformulation of, the EIP without specification change.
+
+Before the EIP has moved out of Draft status and is being versioned, the version number MUST initially have `MAJOR` version `0`.
+
+For every change made to an EIP via a pull request (PR) made to ethereum/EIPs, a new entry MUST be added to the CHANGELOG section of the EIP that outlines the changes made within the PR. This CHANGELOG entry MUST include the following:
+
+1. A new version number that follows the semantic versioning scheme outlined above.
+2. The date when the changes where introduced.
+3. The ethereum/EIPs PR number that implements the changes.
+4. A line for each change made to the EIP's specifications that includes a short description of the change.
+
+Additionally, the new version MUST be added to the metadata header of the EIP's markdown file (to a new "version" field), so that it may be easily parsed.
```

Comment:
>to a new "version" field

This should have higher billing.  Like the first line after the RFCs.  As well as the new changelog section.

```
A new header field `version` is added to new EIPs, as defined below, identifying the  draft and/or review version of the specification.

A new body section "Changelog" is added to new EIPs, as defined below, tracking the changes between draft and/or review versions of the specificaiton.
```

Also, we should update the EIP template to add these two new feaures as part of this EIP when it goes final.